deploy/docker/clickhouse-setup/docker-compose-core.yaml

```diff
@@ -30,9 +30,9 @@ services:
     image: signoz/alertmanager:0.23.0-0.2
     volumes:
       - ./data/alertmanager:/data
-    depends_on:
-      query-service:
-        condition: service_healthy
+    # depends_on:
```

Comment:
Disable dependency on query-service container to allow docker-compose-core to run alone

Reply:
makes sense :+1: